cmake/projects/SPIRV-Tools/hunter.cmake

```diff
@@ -32,13 +32,36 @@ hunter_add_version(
     695e9abf892fe939fbe61398a06b6f177cd43e77
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    SPIRV-Tools
+    VERSION
+    2020.4-p0
+    URL 
+    https://github.com/cpp-pm/SPIRV-Tools/archive/v2020.4-p0.tar.gz
+    SHA1 
+    e408dd86040ba66dbd7cdd386af96cdb11484c44
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    SPIRV-Tools
+    VERSION
+    2020.6-p0
+    URL 
+    https://github.com/cpp-pm/SPIRV-Tools/archive/v2020.6-p0.tar.gz
+    SHA1 
+    16204713db0fc98689d86610c3d7dc29f579b17e
+)
+
 hunter_cmake_args(
     SPIRV-Tools
     CMAKE_ARGS
       BUILD_TESTING=OFF
       SPIRV_SKIP_EXECUTABLES=ON
       SPIRV_SKIP_TESTS=ON
       SPIRV_WERROR=OFF
+      SPIRV_TOOLS_BUILD_STATIC=OFF
```

Comment:
This should be enabled/omitted to prefer static builds https://github.com/KhronosGroup/SPIRV-Tools/pull/3910

Reply:
This is really complicated, so SPIRV_TOOLS_BUILD_STATIC won't actually change static/shared build type, that is still controlled by BUILD_SHARED_LIBS.
Only thing it changes is what ${SPIRV_TOOLS} will be aliased to.
Turning this on will break a lot of hunter changes regarding namespaces, I just got lazy and turned this off.